invenio_db/utils.py

```diff
@@ -11,14 +11,12 @@
 
 from flask import current_app
 from sqlalchemy import inspect
-from werkzeug.local import LocalProxy
 
-from .shared import db
+from .proxies import current_db
+from .shared import db as _db
```

Comment:
I think this was changed in the wrong way?

The previous `db`  should be the same, no change. The previous `_db`, should be the `current_db`:
```suggestion
from .proxies import current_db as _db
from .shared import db
```

Reply:
here i am a little bit confused, because the `from .shared import db as _db` was your [suggestion](https://github.com/inveniosoftware/invenio-db/pull/158#discussion_r1053289837).  The `drop_alembic_version_table` function uses `current_db`. If we change `current_db` to `_db` we would have to change that too in the `drop_alembic_version_table` function. Further, as i understand the two usages should not be the same because in `rebuild_encrypted_properties` the `db` imported from `shared.py` was used and not the `_db` from the `LocalProxy`

Reply:
I am sorry if I made the wrong suggestion.
What I noticed now is that we have switched the imports, which might lead to issues.
Nothing should be changed, given that the imports will be exactly the same as before the changes. The only needed change in this file is the extra param to `rebuild_encrypted_properties`

Reply:
yes, but the new parameter has to come from .shared not from .proxies. they are not the same. but i will investigate a little bit more into that problem.